slides/08_Data_Fetching.html

```diff
@@ -473,14 +279,90 @@ <h2>Solution</h2>
               if (isLoading) return <div>LOADING</div>;
               if (isError) return <div>ERROR while loading data</div>;
             
-              return (
-                <>
-                  {data.results.map((pokemon) => (
-                      <PokeListEntry key={pokemon.name} name={pokemon.name} />
-                    ))}
-                </>
-              );
+              return <PokeList pokemons={data.results} />;
             }
+        </script></code></pre>
+          </section>
+          <section>
+            <p><span class="code">DetailPage.tsx</span></p>
+
+            <pre><code class="hljs tsx" data-trim data-line-numbers="2,3,6|16"><script type="text/template">
+              function DetailPage() {
+                const { pokemonName } = useParams<"pokemonName">();
+                const uri = `https://pokeapi.co/api/v2/pokemon/${pokemonName}`;
+              
+                const { data, isLoading, isError } = useQuery(
+                  ["pokemon", "detail", pokemonName],
+                  () => fetcher<PokemonDetailDto>(uri)
+                );
+              
+                if (isLoading) return <div>LOADING</div>;
+                if (isError) return <div>ERROR while loading data</div>;
+              
+                return (
+                  <div>
+                    <span>{pokemonName}</span>
+                    <img src={data.sprites.front_shiny} alt={pokemonName} />
+                  </div>
+                );
+              }
+        </script></code></pre>
+          </section>
+        </section>
+        <section>
+          <section>
+            <h2>Mutating resources</h2>
+            <ul>
+              <li>
```

Comment:
I assume these bullets are the answer to the question: "Why are we not using the useQuery hook for mutations`, right? Maybe we should put the question onto the slide (in a shorter form)